mavros_msgs/msg/ADSBVehicle.msg

```diff
@@ -0,0 +1,17 @@
+# The location and information of an ADSB vehicle
+#
+# https://pixhawk.ethz.ch/mavlink/#ADSB_VEHICLE
+
+std_msgs/Header header
+
+uint32 ICAO_address
+geographic_msgs/GeoPoint lla
```

Comment:
I'm following ROS conventions. Also: https://github.com/ros-geographic-info/geographic_info/commit/b8043a92e91f38aa09b65b6b31aee96077996bb6#commitcomment-21800531.
So don't see any problem on this. After conversion applied, everything is good